crypto/fipsmodule/service_indicator/service_indicator_test.cc

```diff
@@ -4789,7 +4789,9 @@ TEST_P(KBKDFCtrHmacIndicatorTest, KBKDF) {
                               'D', 'F', '-', 'C', 'T', 'R', ' ', 'K', 'E', 'Y'};
   const uint8_t info[16] = {'A', 'W', 'S', '-', 'L', 'C', ' ', 'K',
                             'B', 'K', 'D', 'F', '-', 'C', 'T', 'R'};
-  uint8_t output[16] = {0};
+  // 14 bytes (112 bits) is the minimum key-derivation key that would be
+  // approved if using an approved algorithm;
+  uint8_t output[14] = {0};
```

Comment:
I'm a little confused. Both `sizeof(secret)` and `sizeof(output)` must be at least 14 for FIPS approval? `KBKDF_ctr_hmac_verify_service_indicator` only checks `secret_len`. Should it also (or instead) be checking `output_len`?

Would it be more appropriate for this PR to shorten the `secret` to length 14 for testing?

Reply:
Ugh yes, at first I had it as output check, then realized the constraint actually refers to the input secret length. Forgot to correct the test subsequently. Will fix here momentarily.